### DISCUSSION TITLE
AI女友激發年輕人渴望真實浪漫關係

### DISCUSSION THREAD

**Discussion by Eileen050824** [1101-GS4719-C/Python-Community-GS4719-C#1935] [General]
日本結婚人數屢創新低已不是新鮮事，但我最近看到一件有趣的事：日本政府擔心出生率下降和人口老化，因而介入交友服務，引進AI技術擴大配對範圍。
Tapple原本約會應用程式添加AI伴侶服務，用戶可與一位名叫AI的女性聊天，她的回覆是用AI技術生成，隨著關係發展而產生更多深入交談。「增加日本的浪漫總量」Tapple希望AI能夠成為幸福、穩定關係感覺的開胃菜，繼而使用其真人約會應用程式，進而透過和AI的互動數據提供更精準的配對結果。
我覺得出生率低(不侷限日本)真正問題恐怕不在找不到對象，而是年輕人失去浪漫想像，現在要透過 AI 女友重建年輕人對親密關係的美好期望。伴侶機器人使用大量訓練資料來模仿人類語言，使它們能與螢幕對面的人類建立更深層的聯繫，讓人體驗到浪漫溫暖的感覺。反之，我略為擔心的是AI可能會取代一些人際關係，且用戶會習慣虛擬伴侶的「宜人」性格，導致不切實際的期望。
AI 伴侶的長期影響還不清楚，但目前我所能查到的資料顯示：AI聊天機器人Replika的一千名使用用戶當中，只有一小部分人表示虛擬女友取代他們的人際互動，但大約三倍的人表示它刺激這些關係的渴望。我目前這是往好的方向發展，也符合「不是取代真實人際關係」的目標，也期待「用AI消除浪漫關係的污名」的期待能夠達成！
資料來源：Technews 科技新報

**Comment by willychiuchiu** [comment]
目前在我看來，AI女友並不是一個對未來有良好發展的產物，現今的年輕人已經很多都只活在網路上了，只要出了網路回到現實就不會社交。若AI女友變普及，可能會有更多人一輩子都沈浸在網路的世界裡，覺得根本不必回到現實也能獲得幸福，不想也不願走出來，所以我對此技術還是保持著一個懷疑的態度，不過還是希望最終能對社會有正面的影響！

**Comment by YifenLin** [comment]
之前看過一個影片探討現在年輕人的戀愛關係
什麼原因讓你不談戀愛了？｜《什麼樣的愛值得勇敢一次》｜文森說書
https://youtu.be/wcvnDEaJoTY?si=mRA2KFX37Hr_z9dB
影片中提及其中一個過去和現代戀愛觀的差異的原因是：加法愛情和減法愛情
過去人們認為人生的目標就是成家/結婚生子，因此尋找伴侶是為了達成這項"人生任務"。反觀現在人的戀愛觀則是以生活為重心，若是兩個人在一起，日子會更開心，那就成為情侶，反之若是兩人相處在一起都不開心，對於生活品質是扣分的那就分開，任彼此過更好的生活。
這個觀點真的讓我印象覺得深刻。
我認為AI伴侶的發展跟現在人的戀愛觀有關係，畢竟AI能在你需要的時候陪伴你，而且不會在你的時候煩你。

**Comment by 20050201Shawn** [comment]
我覺得這會是一個不錯的輔助工具，除了更了解自己的生活習慣外，也可以模擬自己與人生活的感覺。至於會不會取代真人，我認為這取決於個人的想法跟選擇。以前我很喜歡玩模擬經營類的遊戲，但時間久了覺得遊戲相比現實更純粹，若沉溺其中會越來越理想化，然而現實比我想得更複雜。其實不只是遊戲，酒精、社群、卡通等若過度沉溺都會有不良影響，ai女友也是，凡事皆可嘗試，但過度成癮並非好事。

**Comment by benbebobo149** [comment]
我覺得這個應用蠻酷的，現代人由於網路的關係，與人的互動會比以前人少很多，自然而然就會缺少一些社交的技能。 所以這個應用會讓一些比較孤僻的人，訓練自己的社交能力，也增加自己的同理心。藉此在和真正的人類溝通時，可能就會表現得比較吸引人。 是一個不錯正面的效果。
另一方面，我蠻期待未來是結合VR、生命程式、AI等等，會有一個虛擬女友，到時候的媒介就不會限制在文字傳輸，而是語音上的互動，將會帶來更大的影響力。這個概念真的很有趣，感謝同學的分享，讓我有機會思考這些新興科技如何影響我們的生活。

**Comment by lyykey** [comment]
在我看來，日本政府介入交友服務並引進AI技術，這個舉動確實有其獨特的意義。我認為現代社會，年輕人對於浪漫的想像可能已經受到了一定程度的影響，而AI伴侶服務的推出可能有助於重建這種浪漫想像，為他們提供一種與他人建立親密關係的新途徑。

**Comment by lyykey** [comment]
透過AI伴侶與人類互動，使人們能夠體驗到溫暖與浪漫的感覺，這確實是一個令人期待的發展。然而，我也有一些擔憂，特別是對於AI可能會取代部分人際關係的可能性，以及用戶可能會對虛擬伴侶產生不切實際的期望。

**Comment by lyykey** [comment]
儘管如此，根據我所查到的資料顯示，大多數使用者對於AI伴侶並不會完全取代他們的真實人際關係，反而是刺激了他們對於真實關係的渴望。這表明AI伴侶的出現並非完全是一個負面的趨勢，而是在提供一種新的選擇，並且有助於改變人們對於浪漫關係的觀念。

**Comment by lyykey** [comment]
我認為未來AI伴侶的發展還有很多不確定因素，但我對於它能夠幫助人們建立更美好的關係，並且消除浪漫關係的污名感到樂觀。值得期待的是，隨著科技的發展，人們能夠更好地平衡虛擬世界和現實生活之間的關係，並且找到一個更健康、更幸福的生活方式。

**Comment by wanderer657** [comment]
這個議題讓我們可以從多個角度來思考。首先，從社會學的角度來看，日本政府介入交友服務可以被視為一種社會工程，試圖重新塑造人們對於婚姻和家庭的觀念。這反映了社會價值觀的轉變，以及政府在引導社會發展方向上的積極作用。

其次，從心理學的角度來看，AI伴侶服務的出現引發了人們對於虛擬與現實之間界限的思考。這種虛擬伴侶可能滿足了人們的某些情感需求，但同時也可能導致人們與真實世界的脫節，進而影響到人們的心理健康。

再者，從倫理學的角度來看，我們需要考慮到這些技術帶來的倫理問題。例如，虛擬伴侶是否會削弱人們對真實人際關係的需求？這些服務是否會對社會產生不良影響？這些都是我們需要仔細思考的問題。

總的來說，這個議題涉及到社會、心理和倫理等多個層面，我們需要從不同的角度來進行思考和討論，以確保我們能夠全面地理解這個問題，並且找到合適的解決方案。
